src/app/components/forms/community/details/details.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<app-form name="detailsForm">
+		<app-form-group name="name" :label="$gettext(`Name`)">
+			<app-form-control
+				type="text"
+				:rules="{
+					max: 100,
+				}"
+				:disabled="!!formModel.game"
+			/>
+			<app-form-control-errors />
+		</app-form-group>
+
+		<app-form-group name="path" :label="$gettext(`URL Path`)">
+			<app-form-control-prefixed-input
+				prefix="gamejolt.com/c/"
+				data-vv-delay="500"
+				:rules="{
+					pattern: 'urlPath',
+					max: 50,
+					availability: {
+						url: '/web/dash/communities/check-field-availability/path',
+						initVal: model ? model.path : undefined,
+					},
+				}"
+				:disabled="!!formModel.game"
```

Comment:
The prefixed input does not seem to have a "disabled" prop. Which also means that currently, this field is not disabled for game communities even though it should be.

Reply:
Just to be clear, the condition is most likely correct, the prop just doesn't get propagated properly.

Reply:
Prop propagated properly.

Reply:
that made my mind tongue numb.

Reply:
Also, good catch. since the input is no longer top level native attributes cant be applied directly. guess we'll have to keep that in mind.